pkg/cmd/server/api/types.go

```diff
@@ -418,7 +421,7 @@ type ServiceAccountConfig struct {
 
 	// MasterCA is the CA for verifying the TLS connection back to the master.  The service account controller will automatically
 	// inject the contents of this file into pods so they can verify connections to the master.
-	MasterCA string
+	MasterCA *string
```

Comment:
> in case they want to use system roots. missing has to migrate the value we were previously using

In what case would I want to specifically exclude the system roots?  If that's the only reason to have this distinction, then its not very valuable.